doc/Tribler REST API.md

```diff
@@ -243,6 +249,29 @@ Returns a dictionary with the runtime-defined variables that the current Tribler
 }
 ```
 
+## `GET /downloads`
+
+A GET request to this endpoint returns all downloads in Tribler, both active and inactive. The progress is a number ranging from 0 to 1, indicating the progress of the specific state (downloading, checking etc). The download speeds have the unit bytes/sec. The size of the torrent is given in the amount of bytes. The estimated time assumed is given in seconds.
```

Comment:
Myabe list all possible download states. No need for that in the code, but the API needs to have such an enumeration.